### ISSUE TITLE
Is there an option to create a diagram without a database created?

### ISSUE THREAD

**Issue by andresgarcia0313** [dbeaver/dbeaver#2827]
I can not find how to model an ER diagram in dbeaver without other databases

**Comment by serge-rider** [comment]
No, currently DBeaver can work only with existing database entities.

**Comment by sandebert** [comment]
Is this something that is on the roadmap for DBeaver? If so, a rough estimate? 

(I use DBeaver quite a lot - _thanks for a great app!_ - and it sure would be nice to keep my design-db-from-scratch projects in there also without having to resort to another app.)
